lib/player.rb

```diff
@@ -0,0 +1,35 @@
+class Player
+  attr_reader :name, :board
+  
+  def initialize(name)
```

Comment:
What if we initialize a player with a @piece = "X", so then when the player makes a turn their token is already set to "X", we could also set the computer player with a default token of "O"?